test/audits/mobile-friendly/viewport.js

```diff
@@ -18,38 +18,22 @@ const assert = require('assert');
 
 /* global describe, it*/
 
-// Need to disable camelcase check for dealing with background_color.
 /* eslint-disable camelcase */
 describe('Mobile-friendly: viewport audit', () => {
```

Comment:
We should look at the eslint disable comments here, too. I think they're set on by mistake (I made similar mistakes with my test copy-pasta work!)